CONTRIBUTORS.md

```diff
@@ -0,0 +1,92 @@
+# Keptn Contributors
```

Comment:
Wouldn't it make sense to leave this file where it was? Contributions still come in on a repo level. Why move this file to a higher level?

Reply:
I also think this will be hard to maintain.